tests/pages/remoteInstancesPage.js

```diff
@@ -37,6 +39,7 @@ module.exports = {
         usePerformanceSchema: "//input[@name='qan_mysql_perfschema']",
         usePerformanceSchema2: "//input[@name='qan_mysql_perfschema']/following-sibling::span[2]",
         skipTLSL: "//input[@name='tls_skip_verify']/following-sibling::span[2]",
+        skipConnectionCheck: "//[@id='antd']/form/div[4]/label[1]/span[2]",
```

Comment:
Can you use more stable selector here? as i see there is input with name="skip_connection_check" attr

Reply:
Sure! :) I'll look at it :)